samples/csx/HttpSyncStart/function.json

```diff
@@ -0,0 +1,23 @@
+{
+  "bindings": [
+    {
+      "authLevel": "anonymous",
+      "name": "req",
+      "type": "httpTrigger",
+      "direction": "in",
+      "route": "syncorchestrators/{functionName}",
```

Comment:
I wouldn't change the base of the route. `syncorchestrators` makes me think that it's a different kind of orchestrator function, which it's not. Instead, I would add a suffix to the existing one. Something like `"route": "orchestrators/{functionName}/wait"`.

Reply:
Good point!